optimism.tokenlist.json

```diff
@@ -1569,6 +1569,28 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x07FA101efde726E0956Edd2c4D5C8d3d1A5E9c53",
+      "name": "Respawn Finance Wrapped Staked Ethereum",
+      "symbol": "RFWSTETH",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/RFWSTETH.png",
+      "extensions": {
+        "optimismBridgeAddress": "0xeAeAdAC73baaF4cB8B024dE9D65B2eeFa722856C"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x891C46986708b3914A00883B72564AB76817799A",
+      "name": "Respawn Finance Wrapped Staked Ethereum",
+      "symbol": "RFWSTETH",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/RFWSTETH.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
```

Comment:
Same issue here, you are not providing your Kovan contract address and details, only your Optimism Kovan address.